CMakeLists.txt

```diff
@@ -134,7 +134,7 @@ ENDIF()
 
 IF(WITH_VIGRANUMPY)
     set(WITH_BOOST "ON")
-    # set(WITH_BOOST_COMPONENTS ${WITH_BOOST_COMPONENTS} "python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
+    set(WITH_BOOST_COMPONENTS ${WITH_BOOST_COMPONENTS} "python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
```

Comment:
I deliberately don't add `python` as a boost component because I found that the `FIND_LIBRARY` call in `FindVIGRANUMPY_DEPENDENCIES.cmake` is more reliable.